util/chunk/column.go

```diff
@@ -371,6 +377,13 @@ func (c *Column) Float64s() []float64 {
 	return res
 }
 
+// GoDurations returns a Golang time.Duration slice stored in this Column.
```

Comment:
Do we need to add some comment to remind the caller about fsp?

Reply:
More comments are added~